scripts/variantstore/beta_docs/gvs-overview.md

```diff
@@ -93,7 +94,8 @@ This step validates that sample GVCF files contain required annotations and load
 
 GATK tools used: [SplitIntervals](https://gatk.broadinstitute.org/hc/en-us/articles/5358914364699), [GatherVcfsCloud](https://gatk.broadinstitute.org/hc/en-us/articles/5358884598555), [ExtractVariantAnnotations](https://gatk.broadinstitute.org/hc/en-us/articles/9570430059803-ExtractVariantAnnotations-BETA-), [TrainVariantAnnotationsModel](https://gatk.broadinstitute.org/hc/en-us/articles/9570371970587-TrainVariantAnnotationsModel-BETA-), [ScoreVariantAnnotations](https://gatk.broadinstitute.org/hc/en-us/articles/9570326304155-ScoreVariantAnnotations-BETA-), and [GatherTranches](https://gatk.broadinstitute.org/hc/en-us/articles/5358889613339).
```

Comment:
Wait, you use GatherVcfsCloud?  To output shards?  I'm also surprised VETS needs GatherTranches

Reply:
I'll get this clarified, it's possible that this was not updated when we moved to VETS as this was written before then.

Reply:
Confirmed, we do use GatherVcfsCloud when we make a site level VCF to feed to VETS.